src/sphinx_book_theme/assets/styles/base/_base.scss

```diff
@@ -10,6 +10,10 @@
   left: 0;
 }
 
+.d-n {
```

Comment:
Can you just use `d-none` for now since we are inheriting bootstrap's classes anyway?

Reply:
sure thing